components/coding/studentPortal/ProfileDetailCard.tsx

```diff
@@ -21,34 +22,55 @@ function ProfileDetailCard({
   badges,
   totalBadgeCount,
   currentBadge,
-  nextGoal,
+  completedGoal,
+  completedDate,
 }: ProfileDetailCard) {
   return (
-    <div className="flex flex-col h-full gap-4 text-sm border-2 bg-slate-50 text-slate-800 border-slate-800 hover:bg-violet-100">
-      <div className="flex flex-col items-center">
+    <div
+      className="grid grid-cols-4 h-full w-full text-sm border-2 bg-slate-50 text-slate-800 border-slate-800 hover:bg-violet-100"
+      style={{ height: "340px", objectFit: "contain" }}
+    >
+      <div className="flex ml-3 place-items-center col-span-4 mt-4">
         <img
-          className="items-center w-20 mt-4 border-3 rounded-full border-slate-800 "
+          className="col-span-1 w-20 my-3 border-3 rounded-full border-slate-800 "
```

Comment:
another case of relative sizing vs. absolute sizing again

i'd prefer we use something like w-1/4 here, and then a smaller width for the other "circles"

you can always check the tailwind [docs](https://tailwindcss.com/docs/width) to see what width fractions are valid

Reply:
be careful using fractional widths, make sure that it looks good on mobile devices as well. or use tailwind mobile responsive prefixes to look good on mobile. ie `sm:`  or `md:` or `lg:`